### PR TITLE
Purgatory

### PR DESCRIPTION
**Pull request by alexpaxton**
- [ ] Rebased/mergable
  - [ ] Tests pass

Connect #2278 

Skeleton for Purgatory page (name TBD)
User is sent here when they log into an organization and have the role `member`, or no role whatsoever.

Viewable as a (currently) unauthenticated route at `/purgatory`

screen shot 2017-11-07 at 12 45 42 am

### CODE REVIEW

File: ui/src/auth/Purgatory.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const Purgatory = () => {
+  return (
+    <div>
+      <div className="auth-page">
+        <div className="auth-box">
+          <div className="auth-logo" />
+          <div className="auth--purgatory">
+            <h3>
+              Logged in to <strong>[ORGNAME]</strong>
```
Comment (jaredscheib) [inline ui/src/auth/Purgatory.js]: this needs to be a live org name

File: ui/src/index.js [hunk 2]
```diff
@@ -137,6 +142,7 @@ const Root = React.createClass({
         <Router history={history}>
           <Route path="/" component={UserIsAuthenticated(CheckSources)} />
           <Route path="/login" component={UserIsNotAuthenticated(Login)} />
+          <Route path="/purgatory" component={Purgatory} />
```
Comment (jaredscheib) [inline ui/src/index.js]: wrap this in the `UserIsNotAuthenticated` HOC

File: ui/src/auth/Purgatory.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            </h3>
+            <p>
+              You have not been assigned a Role yet<br />Contact your
```
Comment (jaredscheib) [inline ui/src/auth/Purgatory.js]: it's not that you have not been assigned a role, it's that your role lacks sufficient access to see anything.

File: ui/src/auth/Purgatory.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              Administrator for assistance
```
Comment (jaredscheib) [inline ui/src/auth/Purgatory.js]: 1. `Administrator` --> `Admin`
1. `assistance` --> `assistance.`

**Comment by alexpaxton** [comment]
screen shot 2017-11-07 at 11 15 22 pm

Updated screenshot

**Pull request merged**